testsuite/gnat2goto/tests/record_aggregates_box/test.opt

```diff
@@ -0,0 +1,2 @@
+ALL XFAIL gnt2goto Failed precondition from tree_walk.ads
```

Comment:
In CI this seems to pass...

Reply:
I put the erroneous output in the test.out file by mistake.  Gnat2goto should not fail the precondition.  The test is legal Ada.  I have committed a test.out file which closer represents the expected output.

Reply:
OK.